packages/@featherds/dropdown/src/components/FeatherDropdown.vue

```diff
@@ -2,6 +2,7 @@
   <FeatherMenu
     ref="menu"
     class="feather-dropdown-container"
+    data-ref-id="feather-dropdown-container"
```

Comment:
should this be feather-dropdown-menu-container?

Reply:
I did this on purpose to match the class - in this component, this div is the main container for all the content, and there aren't any other ref-id's, so I felt this made sense.

Reply:
...of course, we could do the opposite, and update the class instead?

Reply:
I would rather keep it consistent across components and use the menu-container suffix? We can update the class but I honestly don't mind them being different

Reply:
Resolved in latest